carma_globaer.h

```diff
@@ -187,14 +187,15 @@
 #define gsticki       carma%f_gsticki
 #define gstickl       carma%f_gstickl
 #define cstick        carma%f_cstick
+#define nucl_method   carma%nucl_method
```

Comment:
nucl_method may not be a good term to use as this is only being used for sulfate nucleation and there is already a field (inucproc) that is used to determine the type of nucleation that is used for clouds and aerosols and has values in carma_enums_mod.F90.


 !!  Define values of flag used for nucleation process
  !!  specification (inucproc).
  !!
  !!  NOTE: Some of these can be used in combination, so for aerosol freezing this is treated
  !!  as a bit mask. When setting for one (or more) of the Aerosol freezing methods, use:
  !!    IAERFREEZE + I_AF_xxx + I_AF_yyy + ...
  integer, public, parameter :: I_AF_TABAZADEH_2000 = 1     !! Aerosol Freezing, Tabazadeh[2000]
  integer, public, parameter :: I_AF_KOOP_2000      = 2     !! Aerosol Freezing, Koop[2000]
  integer, public, parameter :: I_AF_MOHLER_2010    = 4     !! Aerosol Freezing, Mohler[2010]
  integer, public, parameter :: I_AF_MURRAY_2010    = 8     !! Glassy Aerosol Freezing, Murray[2010]
  integer, public, parameter :: I_DROPACT           = 256   !! Droplet Activation
  integer, public, parameter :: I_AERFREEZE         = 512   !! Aerosol Freezing
  integer, public, parameter :: I_DROPFREEZE        = 1024  !! Droplet Freezing
  integer, public, parameter :: I_ICEMELT           = 2048  !! Ice Melting
  integer, public, parameter :: I_HETNUC            = 4096  !! Heterogeneous Nucleation
  integer, public, parameter :: I_HOMNUC            = 8192  !! Binary homogeneous gas-to-particle nucleation
  integer, public, parameter :: I_HETNUCSULF        = 16384 !! Binary homogeneous gas-to-particle nucleation